ProjectSettings/EditorBuildSettings.asset

```diff
@@ -4,4 +4,7 @@
 EditorBuildSettings:
   m_ObjectHideFlags: 0
   serializedVersion: 2
-  m_Scenes: []
+  m_Scenes:
```

Comment:
Revert please.

Reply:
I've no idea why these keep getting changed.  Will revert back to those in the dev working branch in a new PR.

Reply:
Resolved in #1648